packages/vercel-edge/src/sdk.ts

```diff
@@ -35,18 +34,14 @@ export const defaultIntegrations = [
 export function init(options: VercelEdgeOptions = {}): void {
   setAsyncLocalStorageAsyncContextStrategy();
 
-  const sdkDefaultIntegrations: Integration[] = [...defaultIntegrations];
-
-  // TODO(v8): Add the request data integration by default.
-  // We don't want to add this functionality OOTB without a breaking change because it might contain PII
-  if (options.sendDefaultPii) {
-    // eslint-disable-next-line deprecation/deprecation
-    sdkDefaultIntegrations.push(new RequestData());
-  }
-
-  if (options.defaultIntegrations === undefined) {
-    options.defaultIntegrations = sdkDefaultIntegrations;
-  }
+  options.defaultIntegrations =
```

Comment:
Aligning this with e.g. the node SDK - otherwise, the `RequestData()` integration is not added to the defaults if somebody extends this (like in nextjs).